docs/licensing.asciidoc

```diff
@@ -28,11 +26,11 @@ EOF
 
 NOTE: You can initiate a trial only if a trial has not previously been activated.
 
-At the end of the trial period, the platinum and enterprise features operate in a link:https://www.elastic.co/guide/en/elastic-stack-overview/current/license-expiration.html[degraded mode]. You can revert to a basic license, extend the trial, or purchase an enterprise subscription.
+At the end of the trial period, the Platinum and Enterprise features operate in a link:https://www.elastic.co/guide/en/elastic-stack-overview/current/license-expiration.html[degraded mode]. You can revert to a Basic license, extend the trial, or purchase an Enterprise subscription.
```

Comment:
We seem to lowercase the license names here https://www.elastic.co/guide/en/elastic-stack-overview/7.5/license-management.html but uppercase them here https://www.elastic.co/subscriptions 😞